gffquant/db/annotation_db.py

```
@@ -180,9 +211,17 @@ def get_features(self, category_id):
     def query_category(self, category_id):
         return self.db_session.query(db.Category).filter(db.Category.id == category_id).one_or_none()
 
+    def get_categories(self):
+        return self.db_session.query(db.Category).all()
+
     @lru_cache(maxsize=10000)
```

Ruff (0.8.0)

Use of `functools.lru_cache` or `functools.cache` on methods can lead to memory leaks

(B019)